pkgs/applications/misc/tuba/default.nix

```diff
@@ -67,20 +70,26 @@ stdenv.mkDerivation rec {
     gst-plugins-base
     (gst-plugins-good.override { gtkSupport = true; })
     gst-plugins-bad
-  ]);
+  ]) ++ lib.optionals clapperSupport [
```

Comment:
If possible, please add a warning that some stuff might be broken. I worked with Rafostar on this and some bug fixes have not made it into a Clapper release yet

Most critical being this one: https://github.com/GeopJr/Tuba/pull/931#issuecomment-2170028583 which will leave Tuba frozen when closing a video.

Reply:
Yeah I've seen that